README.md

```diff
@@ -87,6 +87,34 @@ It is highly recommended to not enable debug info in release mode when building
 debug = false
 ```
 
+## How to release (maintainers only)
+
+physx-rs consists of two crates, physx and physx-sys. If you only merged changes to one of them,
+only that one needs a new release.
+
+1. Look at `physx/CHANGELOG.md` and `physx-sys/CHANGELOG.md` and `api/module/CHANGELOG.md` to determine
+whether both or only one of the crate needs updating, and what [semantic version](https://semver.org/) bump
+we need
+
+2. Review the list of changes in the changelogs and compare with the git commit diffs to the previous release and make sure we've captured and described all changes well and that they are semantically correct
+
+2. Create a branch: git checkout -b release-physx-0.x.y
+
+3. Update relevant changelogs (also update the diff links at the bottom of them!)
+
+4. Make a PR from your branch, get it reviewed and merged. If in doubt about what the PR
+should look like, just look at previous release PRs
+
+5. Once merged, tag the merge with physx-v0.x.y and/or physx-sys-v0.z.w tags as appropriate:
+
+    ```bash
+    git tag physx-v0.x.y -m ""
+    git tag physx-sys-v0.x.y -m ""
+    git push --tags
+    ```
+
+6. Run `cargo publish` in the physx and physx-sys directories as appropriate.
```

Comment:
Think change order of 5 and 6, because of the publish fails for some reason, you'll need to do some commit or PR and then it is easier to have tagged that new commit instead after, instead of having to delete and force-update tags.

Not too common, but does happen.